.github/workflows/build.yml

```
@@ -8,45 +7,50 @@ on:
+
+      - name: Upload Overtime Analyzer EXE as artifact
         uses: actions/upload-artifact@v4
         with:
-          name: 출입기록분석기-exe
-          path: dist/출입기록분석기.exe
+          name: overtime-analyzer-exe
+          path: dist/overtime_analyzer.exe
 
       - name: Create Release
         id: create_release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)